fast64_internal/sm64/sm64_geolayout_bone.py

```diff
@@ -228,27 +237,38 @@ def draw(self, context):
 
         prop_split(col, obj, "geo_cmd_static", "Geolayout Command")
         drawLayerWarningBox(col, obj, "draw_layer_static")
-        col.prop(obj, "use_render_area")
+
+        renderAndShadowBox = col.box()
```

Comment:
I disagree with the use of separate boxes for all of these properties. Properties should be chunked according to their use, and micro splitting each individual feature into its own box looks ugly.
Also selectively adding in the note only after clicking is pretty annoying imo, as it constantly moves the checkbox. I would place all three of these features in a single box, and make the comment permanent.